src/openms/source/ANALYSIS/TOPDOWN/FLASHDeconvAlgorithm.cpp

```diff
@@ -986,19 +987,20 @@ namespace OpenMS
     double mz_bin_max_value = log_mz_peaks_[log_mz_peaks_.size() - 1].logMz;
     Size mass_bin_number = getBinNumber_(mass_bin_max_value, mass_bin_min_value_, bin_mul_factor) + 1;
     bin_offsets_.clear();
-    harmonic_bin_offset_matrix_.clear();
 
     for (int i = 0; i < current_charge_range; i++)
     {
       bin_offsets_.push_back((int)round((mz_bin_min_value_ - filter_[i] - mass_bin_min_value_) * bin_mul_factor));
     }
 
-    harmonic_bin_offset_matrix_.resize(harmonic_charges_.size(), current_charge_range);
+    harmonic_bin_offset_matrix_.getEigenMatrix().resize(harmonic_charges_.size(), current_charge_range);
+    harmonic_bin_offset_matrix_.getEigenMatrix().setZero();
```

Comment:
maybe this setZero is not needed. looks like it overwrites all entries anyway.

Reply:
You are right!! Thanks and all look great! I will also try to apply this in the current branch...

Reply:
k I will remove it here so it is easier for you